src/Packages/RetroMole.Render.Veldrid/Controller.cs

```diff
@@ -759,17 +740,6 @@ private static void SetKeyMappings()
             io.KeyMap[(int)ImGuiKey.Space] = (int)Key.Space;
         }
     }
-    private struct ResourceSetInfo
-    {
-        public readonly IntPtr ImGuiBinding;
-        public readonly ResourceSet ResourceSet;
-
-        public ResourceSetInfo(IntPtr imGuiBinding, ResourceSet resourceSet)
-        {
-            ImGuiBinding = imGuiBinding;
-            ResourceSet = resourceSet;
-        }
-    }

```

Comment:
Hmm.. here bindings and rsi's are comparable in that both hold IDs and the resource
But since we're removing bindings, I believe a triple dict index such as `Sets[Views[Textures[ID]]]` might just work, alternatively we could have `Views` n `Sets` be directly indexed by ID which sounds more convenient but slightly de-couples the dicts... maybe a single `Dictionary<ID, (ResourceSet, TextureView, Texture)>` could work, but again that sounds like a pain